DigitalOffice/Services/TimeManagementService/Database/TimeManagementDbContext.cs

```diff
@@ -18,6 +18,5 @@ public TimeManagementDbContext(DbContextOptions<TimeManagementDbContext> options
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
         }
```

Comment:
I think, no matter do we use Fluent API or not, this line should be here:

modelBuilder.ApplyConfigurationsFromAssembly(Assembly.GetExecutingAssembly());

Anyway, you can argue with me on that. What do you think, should we add it if we don't use it? Common sense tells me not to add it :/

Reply:
Fixed.